### PR TITLE
[fix](alter)(tablet-scheduler) fix unexpected exception with compaction_too_slow message when add rollup for olap table

### LINKED ISSUE #10826: [Bug] (Alter) unexpected  exception throws with compaction_too_slow message when do alter table add rollup operation
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-doris/issues?q=is%3Aissue) and found no similar issues.

### Version

master

### What's Wrong?

create rollup for olap table, and see the following exception trace
```
2022-07-13 18:53:18,169 WARN (doris-mysql-nio-pool-1419|48101) [StmtExecutor.handleDdlStmt():1519] DDL statement(alter table rtb_ad_effects_test add rollup jingmai_rollup (pin_id,business_type,day,date_time,campaign_type,scenario_type,loc,dmp_id,impressions,clicks,cost,landing_page_impressions,bid_price)) process failed.
java.lang.IllegalStateException: COMPACTION_TOO_SLOW
        at com.google.common.base.Preconditions.checkState(Preconditions.java:508) ~[spark-dpp-1.0-SNAPSHOT.jar:1.0-SNAPSHOT]
        at org.apache.doris.alter.MaterializedViewHandler.createMaterializedViewJob(MaterializedViewHandler.java:392) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.alter.MaterializedViewHandler.processBatchAddRollup(MaterializedViewHandler.java:272) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.alter.MaterializedViewHandler.process(MaterializedViewHandler.java:1016) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.alter.Alter.processAlterOlapTable(Alter.java:159) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.alter.Alter.processAlterTable(Alter.java:385) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.catalog.Catalog.alterTable(Catalog.java:5319) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.DdlExecutor.execute(DdlExecutor.java:142) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.handleDdlStmt(StmtExecutor.java:1509) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:436) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.StmtExecutor.execute(StmtExecutor.java:306) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.handleQuery(ConnectProcessor.java:212) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.dispatch(ConnectProcessor.java:349) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.qe.ConnectProcessor.processOnce(ConnectProcessor.java:538) ~[doris-fe.jar:1.0-SNAPSHOT]
        at org.apache.doris.mysql.nio.ReadListener.lambda$handleEvent$0(ReadListener.java:50) ~[doris-fe.jar:1.0-SNAPSHOT]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[?:?]
        at java.lang.Thread.run(Thread.java:834) ~[?:?]
```

### What You Expected?

succeed creating rollup for olap table

### How to Reproduce?

1. set repair_slow_replica to true
2. load data and stop compaction in be
3. for a while when there is a replica in table with compaction_too_slow state and then add rollup for table

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by caiconghui**
# Proposed changes

Issue Number: close #10826 

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (No)
2. Has unit tests been added: (No)
3. Has document been added or modified: (No)
4. Does it need to update dependencies: (No)
5. Are there any changes that cannot be rolled back: (Yes)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java [hunk 1]
```diff
@@ -528,27 +528,24 @@ public int getTabletOrderIdx() {
...
                 chosenReplica = replica;
             }
-            if (replica.getVersionCount() < minVersionCount) {
-                minVersionCount = replica.getVersionCount();
-            }
         }
         boolean recovered = false;
         for (Replica replica : tablet.getReplicas()) {
-            if (replica.isAlive() && replica.tooSlow() && !chosenReplica.equals(replica)) {
+            if (replica.isAlive() && !replica.isBad() && replica.tooSlow() && (!chosenReplica.equals(replica)
```
Comment (weizuo93) [inline fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java:539]: `!replica.isBad()` is redundant.

Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java:539]: alive is not equals to bad？

Comment (weizuo93) [reply fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java:539]: > alive is not equals to bad？

Alive means not bad. 
```
    public boolean isAlive() {
        return getState() != ReplicaState.CLONE
                && getState() != ReplicaState.DECOMMISSION
                && !isBad();
    }
```

Comment (caiconghui) [reply fe/fe-core/src/main/java/org/apache/doris/clone/TabletSchedCtx.java]: ok

**Review by weizuo93 — DISMISSED** [review]
LGTM

**Review by weizuo93 — APPROVED** [review]

**Pull request merged**